style.css

```diff
@@ -0,0 +1,328 @@
+@import url('https://fonts.cdnfonts.com/css/inter');
+
+.first {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+}
+
+.header-first {
+  display: none;
+}
+
+.second {
+  background-image: url(./images/header-illsutration-mobile@2x.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  text-align: center;
+  justify-content: center;
+  flex-direction: column;
+  font-weight: 700;
+  font-size: 30px;
+  color: #172b4d;
+  padding: 12px 12px 12px 15px;
+  font-family: 'Inter', sans-serif;
+}
+
+.second-text {
+  max-width: 511px;
+  margin: auto;
+  font-weight: 400;
+}
+
+.third {
+  font-style: normal;
+  font-weight: 400;
+  text-align: center;
+  line-height: 28px;
+  font-size: 20px;
+  vertical-align: top;
+  color: #172b4d;
+}
+
+.socials {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: flex-end;
+  list-style: none;
+  padding: 5;
+  gap: 20px;
+}
+
+.green {
+  color: #36b37e;
+}
+
+.line {
+  width: 48px;
+  height: 4px;
+  background-color: #36b37e;
+  border-radius: 24px;
+}
+
+.work-section {
+  width: 86%;
+  align-items: center;
+  padding: 10px 15px 15px 10px;
+}
+
+.project-container {
+  display: grid;
+  grid-template-columns: 300px;
+  grid-template-rows: auto auto auto auto auto auto;
+  gap: 20px;
+  justify-content: center;
+  padding: 0;
+}
+
+.project {
+  border: 1px solid green;
+  margin: 0;
+  width: 327px;
+  padding: 0;
+  border-radius: 8px;
+  height: 474px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 16px;
+}
+
+.recent {
+  text-align: center;
+  font-size: 50px;
+}
+
+.image-placeholder {
+  font-family: 'Inter', sans-serif;
+  font-style: 'semi-bold', italic;
+  font-size: 24px;
+  line-height: 32px;
+  line-height: 110%;
+  text-align: center;
+  width: 100%;
+  height: 200px;
+  background-color: #d0d9d4;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+}
+
+.btn {
+  background-color: #36b37f;
+  color: white;
+  width: 122px;
+  height: 48px;
+  margin: 20px 0;
+  font-size: 17px;
+  border: none;
+  border-radius: 5px;
+  transition: 0.8s;
+}
+
+.language {
+  list-style: none;
+  border: 1px solid #ebf0ee;
+  padding: 5px;
+  background-color: #ebf0ee;
+  margin: 0 5px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+}
+
+.lang {
+  display: flex;
+  padding: 0;
+  font-family: 'Inter', sans-serif;
+}
+
+.multi {
+  text-align: center;
+  font-size: 25px;
+  margin: 5px 10px;
+}
+
+.butt {
+  text-align: center;
+}
+
+.dis {
+  display: flex;
+  justify-content: center;
+}
+
+.about {
+  text-align: center;
+  font-size: 30px;
+  display: flex;
+  justify-content: flex-end;
+}
```

Comment:
- [ ] Kindly change the `justify-content` for a `center` value, so it will be responsive on all the sizes of the width


![image](https://user-images.githubusercontent.com/93630700/205401811-d93a3d0d-276f-4021-94ca-97af861707a8.png)